submissions/AriadnaKravchuk/dom/script.js

```diff
@@ -0,0 +1,59 @@
+    },
+    {
+        id: "4",
+        name: "Angus Delaney",
+        illustration: "./assets/img/angus.png",
+        general: 'Angus Delaney is a secondary character of Night in the Woods and Gregg\'s boyfriend. He works the day shift at Video Outpost "Too" and shares an apartment with Gregg in Towne Centre. He is first encountered at band practice and will otherwise be found working the counter at Video Outpost "Too". According to Longest Night, his sign is Fish.',
+        appearance: "Angus is a large, auburn bear with round, thick-rimmed, black glasses. He wears a dark green sweater over an off-white dress shirt with a black tie. He also wears navy pants, black shoes, and a black fedora.",
+        personality: 'Angus is a quiet and polite individual with an interest in science. Bea describes Angus as an "annoyingly strident atheist" and as such, Angus will dismiss the paranormal even when encountering a strange situation. Instead, he prefers an evidence-based approach to almost everything. Compared to most characters, Angus is stoic, shy, and seems rather disconnected from others with the exception of Gregg, though he is deeply concerned about others\' safety and well-being. Bea may remark that Angus is equally a responsible parental figure and boyfriend to Gregg, which she feels is what the latter needs. He is shown to have a good sense of humor when in good company, but can otherwise come off as subdued or joyless. He tends not to let his emotions show, but is rather prone to crying. Angus is also good with computers, to the point where Gregg believes him to be a hacker. He also enjoys cooking to the point where it also functions for him as a coping mechanism in stressful situations. He seems calm enough talking about his past, but some parts of it have clearly stuck with him. He cooks often and likes to bake for other people, due to his mother withholding food from him when he was younger. Angus has mentioned in the supplementary game that even though he is a bear, he does not feel much "Ursidae pride." Angus mentions often feeling like a failure but, is incredibly sensitive about the subject.'
+    }
+]
+
+const navList = document.querySelector(".nav__list");
+const mainInner = document.querySelector(".main__inner");
+const [firstPage] = characters
```

Comment:
```suggestion
const [firstPage] = characters;
```

Use [prettier](https://prettier.io/)